### ISSUE TITLE
Duplicate entries: 'familial hypercholesterolemia' & 'hypercholesterolemia, familial'?

### ISSUE THREAD

**Issue by MichaelaEBI** [EBISPOT/efo#649]
[hypercholesterolemia, familial (MONDO_0007750)](https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0007750) is a child of [familial hypercholesterolemia (EFO_0004911)](https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0004911&viewMode=All&siblings=false) It seems like the Mondo entry is a duplicate since it does not contain much information and all cross-references are contained in the EFO_0004911 entry.

**Comment by paolaroncaglia** [comment]
@MichaelaEBI 
Those two terms don't seem to be the same:
(parent) EFO:0004911 familial hypercholesterolemia
(child) MONDO:0007750 hypercholesterolemia, familial, 1
In Mondo, 'familial hypercholesterolemia' has more than one child, they just weren't all imported in EFO (https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0005439&viewMode=All&siblings=false). 
As for the cross-references of MONDO:0007750, if you believe any of them to be wrong, we'd need to open a ticket in the Mondo tracker. But I will make a note to
- [ ] Check that xrefs of EFO:0004911 familial hypercholesterolemia are correct.
Thanks,
Paola

**Comment by smnorthen** [comment]
@paolaroncaglia
I agree with you that they are not the same as there are different types of familial hypercholesterolemia. So I think it's OK as it is. Can the rest of the Mondo children be imported easily for completeness?

**Comment by paolaroncaglia** [comment]
@smnorthen 
Sure, I will
- [ ] Import MONDO:0011369 hypercholesterolemia, autosomal dominant, 3
- [ ] Import MONDO:0007751 hypercholesterolemia, autosomal dominant, type b

**Comment by smnorthen** [comment]
@paolaroncaglia
Great, thank you!

**Comment by paolaroncaglia** [comment]
@MichaelaEBI 
@smnorthen 
For EFO:0004911 familial hypercholesterolemia, I checked the xrefs added by EFO, and found them all to be correct except for an OMIM one that was moved, so I updated that xrefs. (Details at the bottom of https://github.com/EBISPOT/efo/issues/649#issuecomment-585682454.)
I also imported the two Mondo terms as requested.
All edits will be visible in the mid-March EFO release. Thanks.
